### DISCUSSION TITLE
[SOLVED] Have the option for Browserosaurus to start at macOS launch

### DISCUSSION THREAD

**Discussion by AdrienVillez** [will-stone/browserosaurus#456] [Ideas]
Just because when you select the first link of the day, it takes like 5 to 10 seconds for Browserosaurus to show up.  

:)

**Comment by will-stone** [comment]
Hi @AdrienVillez,

Thanks for the suggestion. I like to rely on OS-level functionality where possible; it means zero added code to maintain, and less to test when upgrading dependencies. Therefore I recommend using `System Preferences > Users & Groups > Login Items` to make sure B starts at login.

<img width="780" alt="image" src="https://user-images.githubusercontent.com/654103/139011303-60330b3e-9682-461a-939e-4f484bfa40d1.png">

Thank you,

Will.

**Reply by AdrienVillez** [reply]
oooooh!  Thank you!  Never knew that was a thing!
